inframanager/api_handler/api_handler.go

```diff
@@ -430,156 +484,161 @@ func (s *ApiServer) SetSnatAddress(ctx context.Context, in *proto.SetSnatAddress
 }
 
 func (s *ApiServer) NatTranslationAdd(ctx context.Context, in *proto.NatTranslation) (*proto.Reply, error) {
-	var err error
-	var podPortIDs []uint16
-	var podIpAddrs []string
-	var serviceEpIpAddrs []string
+	/*
+		var err error
+		var podPortIDs []uint16
+		var podIpAddrs []string
+		var serviceEpIpAddrs []string
+	*/
 
 	out := &proto.Reply{
 		Successful: true,
 	}
-	update := false
+	return out, nil
```

Comment:
For MEV, service code is commented for now as it's not implemented.

Reply:
Based upon target type (can be derived based upon interface type), implementation needs to serve both platforms.

Reply:
Leaving this comment for Venkatest to address as separate commit on this branch.